File: 20113_긴급 회의/20113_긴급 회의_220210.py
from collections import Counter

N = int(input())
votes = Counter(filter(lambda x: x != 0, map(int, input().split())))
count = sorted(votes.items(), key=lambda x: x[1], reverse=True)
if count[0][1] in [x[1] for x in count[1:]]:
    print('skipped')
else:
    print(count[0][0])

File: 20113_긴급 회의/20113_긴급 회의_220210.rb
N = gets.chomp.to_i

votes = gets.chomp.split.map { |v| v.to_i }
vote_table = Array.new(N+1, 0)
for v in votes do 
    vote_table[v] += 1
end
vote_table = vote_table.drop(1)
max_vote = vote_table.max
if vote_table.count(max_vote) > 1
    puts 'skipped'
else
    puts vote_table.index(max_vote) + 1
end
